spring-boot-samples/spring-boot-sample-cache/README.adoc

```diff
@@ -82,11 +82,12 @@ TIP: Run sample cache application using Hazelcast with
 
 
 === Infinispan
-Add the `org.infinispan:infinispan-spring4-embedded` dependency to enable support for
+Add `org.infinispan:infinispan-core` and `org.infinispan:infinispan-spring4-embedded`
+dependencies to enable support for
 Infinispan. There is no default location that Infinispan uses to look for a config
-file so if you don't specify anything it will bootstrap on a hardcoded default. You
-can set the `spring.cache.infinispan.config` property to use the provided
-`infinispan.xml` configuration instead.
+file so if you don't specify anything it will bootstrap on a hardcoded default. However,
+for the sample application to work, `spring.cache.infinispan.config` needs to be set to
+`infinispan.xml` so that necessary caches can be found.
```

Comment:
Oh I get it now. That's because previously you'd have created a "default cache". Yeah I remember that now. Perhaps I should fix that part of the sample differently then. I'll give it some thoughts.